app/controllers/position_imports_controller.rb

```diff
@@ -0,0 +1,7 @@
+class PositionImportsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  def create
+    ImportPositionsJob.perform_now
+    render body: 'Positions will be imported from ICIMs'
+  end
+end
```

Comment:
Curious where this will be used - is it a placeholder? Do you want me to integrate AMS here?